app/src/main/res/values-v21/themes.xml

```diff
@@ -11,6 +11,7 @@
     <style name="AppTheme.NoActionBar.Splash" parent="AppTheme.NoActionBar">
         <item name="android:statusBarColor">@android:color/transparent</item>
         <item name="android:windowTranslucentNavigation">true</item>
+        <item name="android:background">@drawable/bg_splash</item>
```

Comment:
No🙅 
This problem is due to slow processing of `Fragment#inflate`, so it has no effect unless it is a theme setting.
ref: Https://www.bignerdranch.com/blog/splash-screens-the-right-way/